infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesWorkspaceServiceAccount.java

```diff
@@ -108,6 +112,22 @@ private void createExecRole(KubernetesClient k8sClient, String name) {
     k8sClient.rbac().kubernetesRoles().inNamespace(namespace).create(execRole);
   }
 
+  private void createViewRole(KubernetesClient k8sClient, String name) {
```

Comment:
This is kind of at the edges of what I'm particularly familiar with, but how does this work with `createViewRoleBinding()` below? I see that the Kubernetes version creates a cluster rolebinding while the OpenShift version does not mention it.

Reply:
The idea is to get rid of binding a ClusterRole since it turns out OpenSHift users rarely have such privileges.

@sleshchenko may remember why in k8s implementation it was explicitly set to clusterRole while in OpenShift there's no `withKind`

Reply:
I do not remember.
I assume that initially it was done for OpenShift which automatically revolves kind.
And later, when it was ported to Kubernetes - it fails, and kind was added to make it work.